### ISSUE TITLE
When typing a hex color into a resource, the popup prevents keyboard input

### ISSUE THREAD

**Issue by MichaelMacha** [godotengine/godot#80018]
### Godot version

4.0.3

### System information

Linux Mint 21.3

### Issue description

I'm working on a game with items as their own resource class, each of which has a UI color associated with it. While moving from a previous, more haphazard, method of simply including all names, scenes, colors, and so on in parallel arrays; to a more compact version that simply keeps a single array of a custom resource with all necessary properties, I must copy a color from one array to another.

This wouldn't be that big a deal, as I can just copy the hex code for the color; but the moment that the descriptive popup shows up ("Enter a hex code '(#ff0000)' or a named color ('red')") I can no longer enter text with my keyboard until I click off of it. Seems like a UI rules conflict.

Unfortunately, my screen capture key seems to hide the popup, or I would share an example.

### Steps to reproduce

Create any class, extending Resource, with a color as a component. Add the resource to an array in another class, and click on the color, then attempt to type in a hexadecimal code. Note that the popup prevents further input until you remove it.

### Minimal reproduction project

Probably N/A, this should happen for just about any product.

**Comment by AThousandShips** [comment]
Can you replicate this on 4.1.1, or 4.2.dev?

**Comment by tokengamedev** [comment]
Issue of tooltip grabbing focus has been raised multiple times. These issues are collated and fixed under #73926

**Comment by Sauermann** [comment]
That seems right. Closing this as a duplicate of #73926.
